src/core/client/stream/tabs/Comments/PermalinkView/ConversationThreadContainer.tsx

```diff
@@ -129,7 +129,7 @@ const ConversationThreadContainer: FunctionComponent<Props> = ({
                 <IgnoredTombstoneOrHideContainer
                   viewer={viewer}
                   comment={parent}
-                  singleConversationView={true}
+                  allowTombstoneReveal={true}
```

Comment:
Should use the more compact boolean version for consistency:

```suggestion
                  allowTombstoneReveal
```